README.md

```diff
@@ -177,14 +177,12 @@ Your results will then be available in the file codeql_results.csv.
 
 #### Building the browse lists
 ```ruby
-RAILS_ENV=test bundle exec rake browse:all
-RAILS_ENV=test bundle exec rake browse:load_all
+RAILS_ENV=test bundle exec rake browse:all browse:load_all
```

Comment:
@sandbergja did you try this and it worked? I think it needs && between the two commands.